### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by Progress2002**
### Hello Reviewer,

This is @Progress2002  and @shafiuyushawu, and this milestone fulfills the below requirements:

### Project requirements
- Remove `current_user` method in `ApplicationController`, devise will provide us with one.
- [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
  - Can register a new user.
  - User logs in with a combination of email and password.
  - Hashed password should be stored in the database.
  - Ask for confirmation of email.
  - Can reset password.
- [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,24 +1,27 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (t-yanick) [inline app/models/user.rb]: - [x] Great job working on this milestone. As per project requirements, devise is supposed to ask for confirmation of email. In that light, `:confirmable` ought to be enabled in the user model.
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Progress2002,

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work :+1: Commits from both team members
:heavy_check_mark: Devise used.
:heavy_check_mark: Views in devise modified as per requirements
:heavy_check_mark: Use of correct Gitflow



There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Progress2002) [reply app/models/user.rb]: Hello @t-yanick, thanks for the observation, the confirmable feature has been added. Thanks.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You meet all the requested changes by the previous reviewer ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**